pkg/observable/channel/observation_test.go

```diff
@@ -0,0 +1,37 @@
+package channel_test
+
```

Comment:
1. Add a manual comment that this is a helper file for observations
2. You can copy-pasta this whole file into chatgpt and get free inline comments ;)